defold-spine/src/gui_node_spine.cpp

```diff
@@ -500,13 +501,61 @@ static void UpdateBones(InternalGuiNode* node)
     dmVMath::Vector4 scale = dmGui::GetNodeProperty(scene, node->m_GuiNode, dmGui::PROPERTY_SCALE);
     for (uint32_t i = 0; i < num_bones; ++i)
     {
-        spBone* bone = node->m_Bones[i];
         dmGui::HNode gui_bone = node->m_BonesNodes[i];
-
+        spBone* bone = node->m_Bones[i];
         UpdateTransform(scene, gui_bone, bone);
     }
 }
 
+static void FindGuiBones(InternalGuiNode* node, dmGui::HScene scene, dmGui::HNode hnode)
+{
+    // We assume the order is the same as when we created them in the original spine node
+    node->m_BonesNodes.Push(hnode);
```

Comment:
I'm not a fan of this really, but I don't currently see any other options, since a cloned gui node doesn't have an ID set.